repo/packages/E/etcd/0/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "etcd",
+  "version": "0.0.1",
+  "scm": "https://github.com/mesosphere/etcd-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "A distributed consistent key-value store for shared configuration and service discovery.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-small.png",
+    "icon-medium": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-medium.png",
+    "icon-large": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-large.png"
+  },
+  "tags": ["mesosphere", "framework", "consensus"],
```

Comment:
this is ok for multiverse... however we do not want tags for mesos, mesosphere or framework.   This information is used for searches... this information will be universally true and doesn't help with relevancy